### PR TITLE
Rails 5 support

### PR DESCRIPTION
**Pull request by jihaia**
Minimum updates needed to allow Rails 5.0.0.x the ability to include Dynamoid.

### CODE REVIEW

**Comment by tadast** [comment]
FYI I've tried this set of changes and it seems to work well with Rails 5. Not sure why the tests are failing on travis though, seems like a setup issue?

**Comment by jihaia** [comment]
The failing tests occur on master - or at least on the fork I took. Happy to help review and track down the issues, but they appear unrelated to the proposed changes.

**Comment by pboling** [comment]
Tests on master are fixed.  Tests are failing here because you have updated the gems to version that do not work on older versions of Ruby.  Will need to utilize a tool like `appraisals` to make this work across the supported ruby and rails versions.

**Comment by pboling** [comment]
Please rebase on latest master, and fix the spec suite by implementing appraisals.

File: lib/dynamoid/components.rb [hunk 1]
```diff
@@ -23,7 +23,6 @@ module Components
     include ActiveModel::Naming
     include ActiveModel::Observing if defined?(ActiveModel::Observing)
     include ActiveModel::Serializers::JSON
-    include ActiveModel::Serializers::Xml
```
Comment (bglusman) [inline lib/dynamoid/components.rb]: see comments on the other 2 rails 5 fixup PR's :-)

**Pull request closed without merging**